.github/dependabot.yml

```diff
@@ -0,0 +1,26 @@
+# DO NOT EDIT. Generated with:
+#
+#    devctl gen dependabot
+#
+version: 2
+updates:
+- package-ecosystem: gomod
+  directory: "/"
+  schedule:
+    interval: weekly
+    time: "04:00"
+  open-pull-requests-limit: 10
+  reviewers:
+  - giantswarm/team-firecracker
+  ignore:
+  - dependency-name: k8s.io/*
+    versions:
+    - ">=0.17.0"
+- package-ecosystem: docker
+  directory: "/"
+  schedule:
+    interval: weekly
+    time: "04:00"
+  target-branch: master
+  reviewers:
+  - giantswarm/team-firecracker
```

Comment:
Where is this now coming from? Do we have that generated with `devctl`?

Reply:
Added in https://github.com/giantswarm/devctl/pull/75, I thought it would be worth trying it out